AllReadyApp/Web-App/AllReady/Services/AttachmentService.cs

```diff
@@ -7,13 +7,16 @@
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Blob;
 using System.Linq;
+using System.Collections.Generic;
 
 namespace AllReady.Services
 {
     public class AttachmentService : IAttachmentService
     {
         private const string ContainerName = "attachments";
-        public static string[] AllowedExtensions = new string[] { ".jpg", ".gif", ".bmp", ".pdf" };
+        private const int MaxAttachmentBytes = 2 * 1024 * 1024; // 2MB
```

Comment:
is the value for `MaxAttachmentBytes` a blob storage limitation of Azure? If it's not, then 

1. where did you get the value
2. why are we restricting files to this value?

Thanks